### DISCUSSION TITLE
New apps not appears in Overview

### DISCUSSION THREAD

**Discussion by abkein** [Aylur/ags#376] [Q&A]
I just installed Discord, VSCode, etc., but these apps cannot be found from Overview widget.

How to add app to search queries? Is there a folder with app list or are they searched every time in `/bin:/usr/bin`-like folders?

I'm not familiar with JS/TS, so cannot fully understand, how `Appllications.ts` service works, but it seems function `Query` is responsible for this.

**Comment by Aylur** [comment]
It uses `Gio.AppInfoMonitor` as a backend. I don't know about its implementation details, but in theory it should monitor when an application is installed, however it doesn't work in my experience.
It was missing from the wiki, but the service has a `reload` method, which can be called periodically instead, or explicitly after installing a package
```bash
ags -r "Service.import('applications').then(apps => apps.reload())"
```

but this will of course depend on how you implement your widget
